src/compute/leaf.rs

```diff
@@ -111,25 +119,33 @@ pub(crate) fn compute(
         };
 
         let size = node_size.unwrap_or(measured_size).maybe_clamp(node_min_size, node_max_size);
+        let size = size.maybe_max(padding_border.sum_axes().map(Some));
+        // let size = match sizing_mode {
```

Comment:
This commented out code either needs a comment of its own or to be removed.

Reply:
Removed